packages/runtime/container-runtime/api-report/container-runtime.legacy.alpha.api.md

```diff
@@ -96,7 +96,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents
     getCurrentReferenceTimestampMs(): number | undefined;
     getEntryPoint(): Promise<FluidObject>;
     getGCData(fullGC?: boolean): Promise<IGarbageCollectionData>;
-    getGCNodePackagePath(nodePath: string): Promise<readonly string[] | undefined>;
+    getGCNodePackagePath(nodePath: string): readonly string[] | undefined;
```

Comment:
This is API breaking. We aren't allowing any (non-internal) API breakages until `2.3` for (`alpha` APIs).

Reply:
I am just changing a class function here. No one uses these. I understand that "technically" this class is alpha but it isn't really - similar to some of our interfaces as well.
Not allowing to change these will delay things by a lot.

Reply:
Adding some more folks from API council to get their thoughts.

@anthony-murphy @ChumpChief This change is safe and won't break Loop or other customer because they don't extend the container runtime class. The other change in "runtime-definitions.legacy.alpha.api.md" is also safe because it's adding optional params in the data store runtime -> data store context boundary.

Is it okay for this change to merge? The alternative is to sit on this PR for 1+ month which I would want to avoid.